services/blockchain-indexer/shared/dataService/business/token.js

```diff
@@ -105,6 +107,34 @@ const getTokensSummary = async () => {
 	return summary;
 };
 
+const checkTokenAccountExists = async (params) => {
```

Comment:
Please keep the method names consistent.
```suggestion
const tokenHasUserAccount = async (params) => {
```

Reply:
The connector method is tied to the node but the indexer is more separate, flexible and focused on handling user request, hence  had given it a more expressive name. Applied.